src/custom/pages/Claim/state/actions.ts

```diff
@@ -0,0 +1,85 @@
+export const enum ActionTypes {
+  // accounts/address
+  setInputAddress = 'setInputAddress',
+  setActiveClaimAccount = 'setActiveClaimAccount',
+  setActiveClaimAccountENS = 'setActiveClaimAccountENS',
+  // claiming
+  setClaimConfirmed = 'setClaimConfirmed',
+  setClaimAttempting = 'setClaimAttempting',
+  setClaimSubmitted = 'setClaimSubmitted',
+  setClaimedAmount = 'setClaimedAmount',
+  // investment
+  setIsInvestFlowActive = 'setIsInvestFlowActive',
+  setInvestFlowStep = 'setInvestFlowStep',
+  // search
+  setIsSearchUsed = 'setIsSearchUsed',
+  // selected claim rows (table/UI)
+  setSelected = 'setSelected',
+  setSelectedAll = 'setSelectedAll',
+}
```

Comment:
😱

Reply:
in fairness tho, it's TS that adds a lot of bloat, tho it's helpful innit. Maybe i should consider using the createActions here actually 🤔

Reply:
Can we please go for Recoil in the next project 🥺